internal/testdata/snapshots/output/testdata/conflicting_test_symbols/sandbox_unsupported.go

```diff
@@ -0,0 +1,44 @@
+  //go:build !linux
```

Comment:
Little confused as to why a new snapshot file was added here... Is this test not meant to be run on macOS?

Reply:
I've added another tag for macOS for now for similar behavior on macOS and Linux